adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantAnnotationConverter.scala

```diff
@@ -17,6 +17,9 @@
  */
 package org.bdgenomics.adam.converters
 
+import scala.collection.JavaConversions._
```

Comment:
Nit: these should go at the end. Actually, we should lex sort all the imports here, since it should be `htsjdk`, `org.apache`, then `scala`.